core/tracker/inclusion.go

```diff
@@ -18,12 +18,12 @@ import (
 )
 
 const (
-	// inclCheckLag is the number of slots to lag before checking inclusion.
+	// InclCheckLag is the number of slots to lag before checking inclusion.
 	// Half an epoch is good compromise between finality and responsiveness.
-	inclCheckLag = 16
-	// inclTrimLag is the number of slots after which we delete cached submissions.
-	// This matches scheduler trimEpochOffset.
-	inclTrimLag = 32 * 3
```

Comment:
decrease from 3 to 1 epoch since that is max at this point.

Reply:
just wondering, what if we are running charon in a network with `slots_per_epoch` != 32, for example, gnosis? Shouldn't we read this kind of information from beacon node?

If we are taking slot as smallest unit in any network then its fine to hardcode it to 32.